src/phases/victory-phase.ts

```diff
@@ -1,23 +1,46 @@
-import { type BattlerIndex } from "#enums/battler-index";
-import { BattleType } from "#enums/battle-type";
+// -- start tsdoc imports --
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Please use the multi-line disable to make it easier for future developers to add additional tsdoc imports